.github/workflows/ci.yml

```diff
@@ -46,7 +49,7 @@ jobs:
     - uses: actions/cache@v2
       with:
         path: vendor/bundle
-        key: gems-build-rails-${{ matrix.rails_version }}-ruby-${{ matrix.ruby_version }}-${{ hashFiles('**/Gemfile.lock') }}
+        key: gems-build-rails-${{ matrix.rails_version }}-ruby-${{ matrix.ruby_version }}-${{ hashFiles('**/gemfiles/rails_${{ matrix.rails_version }}.gemfile.lock') }}
```

Comment:
I'm not sure this works. :thinking:

Reply:
I think this part probably works alright. I'm inclined to believe it's setting `BUNDLE_GEMFILE` that isn't doing anything.

if caching was failing but the Gemfile was fine, GitHub Actions would try to install `turbo-rails`. I'm inclined to believe setting the Gemfile doesn't work, but caching still does, because there's no mention of `turbo-rails` or `tailwindcss-rails` in the logs, and the logs still say they're `using` a gem that has already been downloaded.

Reply:
`tailwindcss-rails` now appear in the logs of the latest runs.

Reply:
It doesn't look like the call to `hashFiles` is returning anything.